shopinvader/services/sale.py

```diff
@@ -23,6 +23,15 @@ def get(self, _id):
     def search(self, **params):
         return self._paginate_search(**params)
 
+    def ask_email_invoice(self, _id):
+        """
+        Ask to receive invoices related to sale ID by email
+        :param _id: int
+        :return:
+        """
+        self.ask_email_invoice = True
```

Comment:
@acsonefho Why do you assign `True` to your method?

Reply:
@lmignon It's a mistake! I put the same name but it shouldn't be the same! Thank you :)